blog/_posts/2017-09-18-story-of-an-issue.md

```diff
@@ -0,0 +1,100 @@
+---
+title: A story of an issue  
+layout: post
+created_at: Mon Sept 18 2017
+permalink: blog/2017-09-18-story-of-an-issue
+current: blog
+author: Neta and Rete
+twitter: ruby_bears
+categories:
+- student-posts
+---
+
+# A story of an issue
+
+**No matter if the issue is big or small, complicated or simple, a bug or a feature - in our Summer of Code the issues have a similar cycle. Team "Code Bears" has some important lessons to share after two and a half months of working on [diaspora\*](https://diasporafoundation.org/).**
+
+#### Pick out the issue
+
+What do we want to achieve? What would we like to learn? Do we want to use skills we already learned or learn something new? Would we actually see the results of our work? We try to ask ourselves all those questions when we approach the long [list of issues](https://github.com/diaspora/diaspora/issues) of diaspora\*. Naturally at first we got a lot of direction from our coaches and mentor on this step, but as the summer progressed we had stronger opinions on those questions ourselves.
+
+For instance, the more back-end the better. We just love our RoR programming! Of course we also love it when we learn and understand front-end code, but it's very clear that we both are more attracted to the back-end side.
+
+#### Ask the diaspora\* community
+
+Diaspora\* has a very active community and we had to find out for each issue we chose if it was still available. We got great feedback from the community just by asking about it on the issue page. In [some cases](https://github.com/diaspora/diaspora/issues/7471) it even triggered a long discussion about ways to solve the issue or even the very necessity of a solution. The community is very responsive whenever we have questions or require consultation on our solution. You just gotta love the diaspora\* community for their involvement and investment.
+
+#### Search. Everywhere.
+
+Like two Alices in diasporaland, with every issue we embark on an ongoing quest through the intricate and entangled maze called the diaspora source code. It is not your typical MVC app (if that even really exists). Finding the related files to the issue and trying to understand the relationship between them is no less than a superpower one needs to develop.
+
+Apart from the familiar models, controllers and views, we have encountered, and thereby learned about, lots of other types of files: mailers, presenters, helpers, services, workers, templates, javascript files, specs, factories and more.  
+<div style="text-align:center"><img src ="img/blog/2017/2017-09-18-codebears1.gif" style="width: 350px;" /></div>
```

Comment:
All images/gifs are missing image credits. Also, all of them breaks for some reason. Can you use image-credits style and fix all of them?